src/App.js

```diff
@@ -796,6 +796,12 @@ export class App extends React.Component<AppProps, AppState> {
                         case("selectRight90"):
                             this.setState({ "selectedAction": "right90" });
                             break;
+                        case("selectRight180"):
+                            this.setState({ "selectedAction": "right180" });
+                            break;
```

Comment:
We have removed turn right 180 action and don't need the "selectRight180" shortcut anymore.